src/script_chainer/win_exe/script_runner.py

```
@@ -0,0 +1,327 @@
+
+    #     print_message('5秒后关闭本窗口')
+    #     time.sleep(5)
+    finally:
+        # # 清理Push资源
+        # global _push_instance
+        # if _push_instance is not None:
+        #     try:
+        #         _push_instance.ctx.after_app_shutdown()
+        #     except Exception as e:
+        #         log.error(f'清理Push资源失败: {e}')
+        return
```

Ruff (0.14.7)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)